src/descformats/handle.py

```diff
@@ -0,0 +1,200 @@
+import os
+import copy
+
+class DataHandle:
```

Comment:
Mainly it is just an attempt to avoid having to change a bunch of stuff because the base class names changed.  I.e., everything in TXPipe is expecting a DataFile and pretty much just uses the open() method, while everything in RAIL is expecting a DataHandle and uses some combination of iterator(),  initialize_write(), write_chunk() and finialize_write() but it would be nice to have a base class that defines the interface.   Having DataFile inherit from DataHandle seemed like one way to deal with this.

Reply:
Could you put something in the docstring to make this clear?